hazelcast/src/test/java/com/hazelcast/cluster/ClusterJoinConfigCheckTest.java

```diff
@@ -52,28 +51,6 @@ public void killAllHazelcastInstances() throws IOException {
     }
 
     @Test
-    public void tcp_whenGroupPasswordMismatch_thenNewNodeIsShutDown() {
```

Comment:
Not super-psyched about the test I put in, since its not going to be as reliable as we would want it to be, but its something.